proposals/incubation/keptn.md

```diff
@@ -0,0 +1,163 @@
+* [Grafana](https://github.com/keptn-sandbox/grafana-service): Integration to automatically create dashboards for services managed by Keptn. 
+* [Ansible](https://github.com/keptn-sandbox/ansibletower-service): Integration to trigger Ansible Tower playbooks as part of Keptn's orchestrated remediation sequences. 
+* [Jenkins](https://github.com/keptn-sandbox/keptn-jenkins-library): Jenkins shared library for integrating Keptn use cases with Jenkins pipelines.
+* [Auto-remediation working group](https://docs.google.com/document/d/1_WlLP6oLcHe0yyC7kXH2hB3i9bOPvIArp83NohE78FU/edit#): Keptn maintainers initiated a working group to discuss the current state and requirements of auto-remediation and to define the future of automated operations where auto-remediation is a big part of.
+
+Integrations that are currently planned for Keptn, in no particular order:
+
+* [ArtifactHub](https://github.com/keptn/keptn/issues/3406) to provide a central hub for Keptn itself and Keptn integrations. Currently, integrations are curated via GitHub, and the ArtifactHub can provide a holistic view for the user on all available integrations.
+* [OpenTelemetry](https://github.com/keptn/enhancement-proposals/pull/30) to align the Keptn's internal "Keptn context" with the trace-context supported by OpenTelemetry.
+* [ArgoCD](https://github.com/keptn-sandbox/argocd-service/issues/1) for deployments as an alternative via Helm charts and ArgoRollouts that are already supported by Keptn.
+* [Flux](https://github.com/keptn/keptn/issues/2669), similar to ArgoCD, for deployments as an alternative via Helm charts and ArgoRollouts that are already supported by Keptn
+* [Falco](https://github.com/keptn-sandbox/falco-service/issues/1) to identify any security issues and have its ruleset managed by Keptn and use Keptn to orchestrate counter-action in response to identified security threats.
+* [Vault](https://github.com/keptn/keptn/issues/4342) to support encrypted secret management in Keptn.
+* [Snyk](https://github.com/keptn/keptn/issues/417) to be integrated into the CD process to check for security vulnerabilities of container images as part of a quality gate and as a dedicated security scanning task during multi-stage delivery.
+
+
+## Incubation Stage Requirements
+
+### Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope
+
+1. [Schlumberger](https://slb.com/), an oilfield services company working in more than 120 countries, has currently 4 applications evaluated with Keptn quality gates. Evaluations make use of 10-20 SLIs, one of them even 90 SLIs per quality gate evaluation. Their integration triggers Keptn quality gates via Azure DevOps automation daily. It has been [presented it in a Keptn user group](https://youtu.be/9YYjk1e9ZGE).
```

Comment:
Re Schlumberger - According to the definition of [end-user](https://github.com/cncf/toc/blob/main/FAQ.md#what-is-the-definition-of-an-end-user), I’m not sure Schlumberger qualifies under "but do not sell any cloud native services externally", given they are also a Software vendor: https://partners.amazonaws.com/partners/001E000000xHbWRIA0/Schlumberger   https://www.slb.com/newsroom/press-release/2021/pr-2021-0629-slb-ibm-osdu  

I emailed `info@cncf.io` and will verify with the TOC if that's the case.

Reply:
I verified and this submission has at least three other companies that I would qualify as end users, so the project is still fulfilling this requirement.

Reply:
Verified that Schlumberger here qualifies as an end user. 👍

Reply:
I've responded but just to follow up here, the definition of end-user is only applied for CNCF members.
Here, what's needed is *adopters* of the project, and the video's a good example.